RELEASE_NOTES.md

```diff
@@ -81,19 +81,23 @@ you are strongly advised to take a backup first.
   along with the traditional color-picker for manual sampling of neutral colors.
 
   Color calibration internally implements gamut-mapping and gamut-clipping,
-  which attempts to preserve the hue and luminance of pixels while avoiding imaginary and
-  out-of-gamut colors at the beginning of the pipeline, to improve the robustness of
+  which attempt to preserve the hue and luminance of pixels while avoiding imaginary and
+  out-of-gamut colors at the beginning of the pipeline, in order to improve the robustness of
   color-grading modules inside the pipeline. This is made necessary because
   white balancing will only push input out-of-gamut colors farther away,
-  and will noticeably help dealing with artifical colored lights (LED, stage lighting, etc.).
+  and gamut-mapping and -clipping will noticeably help dealing with artifical colored
+  lights (LED, stage lighting, etc.).
 
   A new workflow setting "chromatic adaptation defaults", in processing preferences,
   allows users to choose to keep using the white balance module
   to perform the chromatic adaptation for new edits ("legacy" worflow, the default)
   or to use the new color calibration instead ("modern" workflow).
   The modern workflow still uses the usual white balance module, although
   with different default settings, because input color profiles and
-  demosaicing need at least a rough white balancing early in the pipe.
+  demosaicing need at least a rough white balancing early in the pipe. The new color
+  calibration necessitates the use of the standard color matrix as the
+  input color profile, and the use of the 'camera referred' setting in the white
+  balance module.
```

Comment:
This is my understanding, may be wrong.

Reply:
should be `demoasicing needs`

Reply:
I was referring to Aurélien's text from https://discuss.pixls.us/t/introducing-color-calibration-module-formerly-known-as-channel-mixer-rgb/21227:
>Then, the default parameters of the color calibration will:
> - read the raw EXIF to fetch the scene white balance set by the camera,
> - adjust this setting with the reference white balance used in white balance module,
> - adjust this setting again with the input color profile in use (**only the standard matrix**).

I may have misunderstood.